kernel/data-pipeline/api/src/main/java/org/apache/shardingsphere/data/pipeline/spi/sqlbuilder/PipelineSQLBuilder.java

```diff
@@ -150,6 +150,14 @@ default Optional<String> buildCreateSchemaSQL(String schemaName) {
      */
     String buildSplitByPrimaryKeyRangeSQL(String schemaName, String tableName, String primaryKey);
     
+    /**
+     * Build select offset SQL.
+     * @param schemaName schema name
```

Comment:
1, It's better to add newline between javadoc description and params.

2, Consider to rename method. It's better not use offset when there's no unique key.